conda_forge_tick/update_upstream_versions.py

```diff
@@ -189,7 +189,7 @@ def get_latest_version(meta_yaml, sources):
     return False
 
 
-def update_upstream_versions(gx, sources=(PyPI(), CRAN(), Github(), RawURL())):
+def update_upstream_versions(gx, sources=(PyPI(), CRAN(), RawURL(), Github())):
```

Comment:
Will this increase execution times?

Reply:
Probably. Are the 4 scripts run separately or all at once. If separately, I don't think it'll be a huge issue.

Reply:
They are currently run in sequence.